### DISCUSSION TITLE
Can't move widgets MacOS Monterey

### DISCUSSION THREAD

**Discussion by haydenbroadhead** [johnlindquist/kit#1184] [Q&A] [answered]
I'm not sure if this is  a setting i need to enable or something, but when i try to move a widget (e.g. the corgi script) it doesn't move the window but drags the contents of the image.

**Comment by johnlindquist** [comment, marked as answer]
@haydenbroadhead 

There's a special webkit property that will make a dom element drag its containing app:

```js
await widget(`<img
style="-webkit-app-region: drag;"
src="https://cdn.midjourney.com/940b046a-818e-49ed-b67e-f200e3607220/0_0.png">`)
```

> Note: This wasn't required a few months back. The behavior of `-webkit-app-region` and how it deals with other mouse events changed this year

**Reply by haydenbroadhead** [reply]
ah great, works as i need now. Thanks John :)

**Reply by Strajk** [reply]
omg what is this sorcery, but it works!
